great_expectations_cloud/agent/agent.py

```diff
@@ -133,6 +140,8 @@ class GXAgent:
     _PYPI_GREAT_EXPECTATIONS_PACKAGE_NAME = "great_expectations"
 
     def __init__(self: Self):
+        self._config = self._create_config()
```

Comment:
I was hoping to get the config creation out of the constructor, but some existing testing and error messaging relies on it, and is out of scope for this PR.